### PR TITLE
Official 1.14 Release Docs

### PR DESCRIPTION
**Pull request by jimangel**
### 🛑 DO NOT MERGE 🛑 UNTIL KUBERNETES 1.14 IS RELEASED
Target Date: 2019-03-25

## Release deadlines

From [sig-release](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.14):

| **Date** | **Description** |
| --- | --- |
| MAR 1  | Open placeholder PRs
| MAR 11 | PRs ready for review
| MAR 18 | All PRs reviewed and ready to merge

## More information
Contact: @jimangel @MAKOSCAFEE @cody-clark @jaredbhatti @npentrel @simplytunde with any question.

### CODE REVIEW

**Comment by jimangel** [comment]
## Release 1.14 megabranch

PR | Based on `dev-1.14` | Tech LGTM | Docs approved | Merged
---|---|---|---|---
|https://github.com/kubernetes/website/pull/12929 |Y | N| N| N |
|https://github.com/kubernetes/website/pull/12915 | Y | Y | Y | N|
|https://github.com/kubernetes/website/pull/12297 | Y| N | N | N |


Edit: [WIP]

File: content/en/docs/setup/independent/install-kubeadm.md [hunk 1]
```diff
@@ -79,10 +79,28 @@ The pod network plugin you use (see below) may also require certain ports to be
...
+| Docker     | /var/run/docker.sock             |
+| containerd | /run/containerd/containerd.sock  |
+| CRI-O      | /var/run/crio/crio.sock          |
+
+If both Docker and containerd are detected together, Docker takes precedence. This is
+needed, because Docker 18.09 ships with containerd and both are detectable.
+If any other two or more runtimes are detected, kubeadm will exit with an appropriate
+error message.
+
+On non-Linux nodes the container runtime used by default is Docker.
```
Comment (sftim) [inline content/en/docs/setup/independent/install-kubeadm.md]:
```suggestion
On non-Linux nodes, kubeadm defaults to using  Docker as the container runtime.
```

(My suggestion, however original wording is OK too).

File: content/en/docs/setup/independent/install-kubeadm.md [hunk 2]
```diff
@@ -79,10 +79,28 @@ The pod network plugin you use (see below) may also require certain ports to be
...
+
+If the container runtime of choice is Docker, it is used through the built-in
```
Comment (sftim) [inline content/en/docs/setup/independent/install-kubeadm.md]:
```suggestion
When using Docker as the container runtime, Kuberenetes relies on a built-in
```

File: content/en/docs/tasks/extend-kubectl/kubectl-plugins.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ content_template: templates/task
 
 {{% capture overview %}}
 
-{{< feature-state state="beta" >}}
+{{< feature-state state="stable" >}}
```
Comment (sftim) [inline content/en/docs/tasks/extend-kubectl/kubectl-plugins.md]:
```suggestion
{{< feature-state for_k8s_version="1.14" state="stable" >}}
```

File: content/zh/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 4]
```diff
@@ -489,7 +489,7 @@ pod "web-2" deleted
 
 观察 StatefulSet 的 Pod，等待它们全部变成 Running 和 Ready。
 
-```
+```shell
```
Comment (sftim) [inline content/zh/docs/tutorials/stateful-application/basic-stateful-set.md]: This code block mixes a command and some output (we should fix that to use the official style in a separate PR).
I don't think it's right to mark this as POSIX shell code.

File: content/zh/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 5]
```diff
@@ -480,7 +480,7 @@ web-2   k8s.gcr.io/nginx-slim:0.8
 `web-0` has had its image updated, but `web-0` and `web-1` still have the original 
 image. Complete the update by deleting the remaining Pods.
 
-​```shell
+```shell
```
Comment (sftim) [inline content/zh/docs/tutorials/stateful-application/basic-stateful-set.md]: I'd expect that we didn't change the _zh_ locale documents in this PR. Have I got that right?

**Pull request merged**